docs/GHA.md

```diff
@@ -12,17 +12,3 @@ In a GitHub Actions workflow, you can set these environmental variables using th
       core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
       core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
 ```
-
-To write to the cache, set `SCCACHE_GHA_CACHE_TO` to a cache key, for example
-`sccache-latest`. To read from cache key prefixes, set `SCCACHE_GHA_CACHE_FROM`
-to a comma-separated list of cache key prefixes, for example `sccache-`.
-
-In contrast to the [`@actions/cache`](https://github.com/actions/cache) action, which saves a single large archive per cache key, `sccache` with GHA cache storage saves each cache entry separately.
-
-GHA cache storage will create many small caches with the same cache key, e.g. `SCCACHE_GHA_CACHE_TO` and `SCCACHE_GHA_CACHE_FROM`. These GHA caches are differentiated by their [_version_](https://github.com/actions/cache#cache-version). The GHA cache implementation in `sccache` calculates the cache version from the [`sccache` entry key](docs/Caching.md), e.g. the source file path.
```

Comment:
Removing the cache from/to options makes it impossible to configure the cache to write-only. Write-only cache is useful for allowing PR builds to use the main branch cache, but forcing the main branch build to build without caching.

Reply:
Write-only seems useful. Can you raise an issue of this feature? 

We can implement the same cache_to and cache_from (if needed) by using two [ghac](https://opendal.databend.rs/opendal/services/ghac/index.html) clients with different versions.

Also, [OpenDAL](https://github.com/datafuselabs/opendal) is open to any contributions. Welcome for any comments and discussion :heart: